HelpMyStreetFE/HelpMyStreetFE/Controllers/RequestHelpApiController.cs

```diff
@@ -23,6 +24,26 @@ public RequestHelpAPIController(ILogger<RequestHelpAPIController> logger, IReque
             _requestService = requestService ?? throw new ArgumentNullException(nameof(requestService));
         }
 
+
+        [Authorize]
+        [HttpPost("accept-request")]
+        public async Task<ActionResult<bool>> AcceptRequest([FromBody]UpdateJobRequest Job)
+        {
+            try
+            {
+                var userId = int.Parse(HttpContext.User.FindFirst(ClaimTypes.NameIdentifier).Value);
+                int jobId = -1;
+                int.TryParse(Base64Utils.Base64Decode(Job.JobID.Replace("job-", "")), out jobId);
```

Comment:
Not sure why I appended `job-`. Might be easier to remove this in html etc.

Reply:
Certainly the back-end should have no idea that this is what's happening in the front end, so `job-` should really be taken off on the front-end code